results/evaluation/bart-test-extrinsic-100-summaries.json

```diff
@@ -1542,7 +1542,7 @@
     "fbs_classifier_summary": "The former RT\u00c9 broadcaster, M\u00e1irt\u00edn \u00d3 Murchun, has died in Dublin.",
     "fbs_oracle_count_extrinsic": 0,
     "fbs_oracle_has_failed": false,
-    "fbs_oracle_has_unknown": true,
+    "fbs_oracle_has_unknown": false,
```

Comment:
this was already false on `main` -- maybe this diff will go away on a rebase against main?

Reply:
weird, when trying to rebase:
```
git rebase origin/main
Current branch fix-pegasus is up to date.
```